pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/preprovisioned/index.md

```diff
@@ -24,7 +24,7 @@ If `keepalived` is enabled, update Konvoy 1.8 Configuration with the interface u
     ens192
     ```
 
-1.  Update `cluster.yaml` with the output of the previous commanddocs/site/major-version-upgrade/upgrade/prepare-local-environment/index.md:90
+1.  Update `cluster.yaml` with the output of the previous command:
```

Comment:
I think this is supposed to be a link to.... somewhere.

Reply:
I actually looked through this - I think was a misfire from copy and paste from before.